charts/engine/templates/engine-job-template-gpu-cm.yaml

```diff
@@ -37,6 +37,12 @@ data:
 {{ if .Values.schedulerName }}
           schedulerName: {{ .Values.schedulerName }}
 {{ end }}
+{{ if .Values.tolerations }}
```

Comment:
Let's just rename it to add gpu tolerations

Reply:
done